IRO-1674 - Remove emotion

brekk opened the pull request:
Though `@chakra-ui/react` uses `emotion` under the hood, we should be taking advantage of the style / sx props to do things now.

This PR removes all references to `emotion` except in `package.json` (needed for tests / a dependency of `@chakra-ui/react`) and in `components/IronFishUIProvider.tsx` which is using emotion's `Global` component to provide fonts.

I removed a bunch of pre-existing components for Forms since we will wanna use chakra's approach instead. I also created two simple stories for Colors / Swatches.

brekk commented on .husky/pre-commit:
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+nps care
```
Added this, you should be able to get it by running:
1. `yarn`
2. `npx husky install`

ndemidovich-sc commented on components/Colors.tsx:
```diff
@@ -1,13 +1,12 @@
-/** @jsxImportSource @emotion/react */
 import type { ReactNode } from 'react'
```
Do we need this file at all?
If you want to leave this code as exampe, then move it into stories.

ndemidovich-sc commented on components/Showcase.tsx:
```diff
@@ -12,19 +9,17 @@ export type ShowcaseProps = {
+        background={headerColor}
+        p="4rem 2rem"
+        w="calc(100% - 4rem)"
+        textAlign="left"
```
Again, if it need as example, please move it into stories.

ndemidovich-sc commented on components/Swatch.tsx:
```diff
@@ -46,74 +45,48 @@ export const Swatch = ({ name, hex }: SwatchProps) => {
+      overflow="hidden"
+      justifyContent="space-between"
+      alignItems="center"
+      color={blackOrWhite(hex)}
```
Similar as in previous comments.

ndemidovich-sc commented on package-scripts.js:
```diff
@@ -37,9 +33,14 @@ module.exports = {
-    build: 'tsc --project tsconfig.json',
+    build: {
+      description: 'build the codebase with tsc',
+      clean: rimraf('dist'),
```
Great. I wanted to do it in the next task. 😃

azhurauski-sc commented on svgx/darkmode.tsx:
```diff
@@ -0,0 +1,16 @@
+export default function DarkMode() {
```
There is a components with the same name in chakra-ui to [force dark or light mode](https://chakra-ui.com/docs/styled-system/features/color-mode#forcing-a-specific-mode), so maybe it could be called like NightIcon, DarkModeIcon, (MoonIcon exists in chakra icons as well)

brekk commented:
@ndemidovich-sc I moved all the mentioned components to `/stories` instead.
@azhurauski-sc I renamed that mentioned component to `IconDarkMode` and did the same to the light mode component.

ndemidovich-sc commented on stories/Colors.tsx:
```diff
@@ -26,78 +26,52 @@ const ColorStack = ({
-  </div>
-)
+      <H1 textAlign="left">{title}</H1>
+      <chakra.div
```
I think you can use component `Box` instead of this structure

ndemidovich-sc reviewed (approved):
Minor remark.
